DQM/GEM/plugins/GEMDQMHarvester.cc

```diff
@@ -178,10 +194,13 @@ void GEMDQMHarvester::createSummaryVFAT(edm::Service<DQMStore> &store,
   copyLabels(h2Src, h2Sum);
 }
 
-Int_t GEMDQMHarvester::assessOneBin(Float_t fAll, Float_t fNumOcc, Float_t fNumWarn, Float_t fNumErr) {
-  if (fNumErr > 0.05 * fAll)  // The error status criterion
+Int_t GEMDQMHarvester::assessOneBin(
+    std::string strName, Int_t nIdxX, Int_t nIdxY, Float_t fAll, Float_t fNumOcc, Float_t fNumWarn, Float_t fNumErr) {
```

Comment:
What is the role of ```strName``` here? It is not used in this function.

Reply:
Yes, they are not used currently, but I let them leave for future usage.